code/game/objects/items.dm

```diff
@@ -319,11 +319,65 @@
 	..(user, size, show_name)
 	if(price && price > 0)
 		to_chat(user, "You read '[price] space bucks' on the tag.")
-	if((cant_drop != FALSE) && user.is_holding_item(src)) //Item can't be dropped, and is either in left or right hand!
-		to_chat(user, "<span class='danger'>It's stuck to your hands!</span>")
+	if(user.is_holding_item(src))
+		if(reagents.total_volume)
+			//If the examiner is holding the item and can feel that the item contains hot or cold reagents, indicate that the item feels hot or cold.
+			var/can_feel_temperature = FALSE
+			var/nopain = FALSE
+			if(isliving(user))
+				can_feel_temperature = TRUE
+				var/mob/living/L = user
+				if(!L.feels_pain() || L.has_painkillers())
+					nopain = TRUE
+			//If we have something covering our hands, we can't feel the searing heat of the beverage we're about to enjoy.
+			if(iscarbon(user))
+				var/mob/living/carbon/C = user
+				if(C.check_body_part_coverage(HANDS))
+					can_feel_temperature = FALSE
+			if(can_feel_temperature)
```

Comment:
maybe also add an `else "you can't feel its temperature"`

Reply:
Perhaps bluespace or no-react beakers should have that, since you could argue that the bluespace one uses alt dimension tech (bluespace) to store extra reagents, so you would not be able to feel anything. Same with the no-react ones. It's stored in stasis, so all the energy on the reagent is also maintained in the dimensional stasis, so you can't feel anything either.

Reply:
well i was thinking more along the lines of "letting gloved spessmen know they can't feel the beaker because they're wearing gloves"

Reply:
Well, most gloves would transfer heat quite well. Latex gloves, assorted gloves, botany gloves, etc. Even the insulos could be argued to let heat through, albeit slower due to being rubber/whatever.

Reply:
In that case every held item would have to have such a message on every examine, lest you give away that there is some, unknown temperature extreme.

One thing I was considering was having gloves simply limit it to the 'it feels warm/cold' even if it's at the dangerous levels. But then yeah there are considerations for the thermal conductivity of the specific glove and stuff.